docs/general/basic_usage.rst

```diff
@@ -2,63 +2,75 @@
 
-You could simply make your application assume that Aldryn Boilerplate Bootstrap 3 will be available. That's not ideal
-though, because it will be off-putting to people who don't want to have to use it. A reusable application should have
-requirements that are as generic as possible, not based on a particular frontend framework.
+You could simply make your application assume that Aldryn Boilerplate
+Bootstrap 3 will be available. That's not ideal though, because it will be
+off-putting to people who don't want to have to use it. A reusable application
+should have requirements that are as generic as possible, not based on a
+particular frontend framework.
 
-So, although it means a little more work for you, you should **also** provide more generic frontend (templates, CSS
-etc) support for the application, and if you like, for other Boilerplates too.
+So, although it means a little more work for you, you should **also** provide
+more generic frontend (templates, CSS etc) support for the application, and if
+you like, for other Boilerplates too.
 
-At the very least, the developers who use your application will find it easier to create templates and static file for
-it that support their own frontend conventions if they they can start with simple ones.
+At the very least, the developers who use your application will find it easier
+to create templates and static file for it that support their own frontend
+conventions if they they can start with simple ones.
 
 
 Aldryn Boilerplates
 ===================
 
-To make this easier, use the `Aldryn Boilerplates <https://github.com/aldryn/aldryn-boilerplates>`_ application.
+To make this easier, use the
+`Aldryn Boilerplates <https://github.com/aldryn/aldryn-boilerplates>`_
+application.
 
-This provides support for multiple Boilerplates, allowing you to offer rich frontend machinery compatible with Aldryn
-Boilerplate Bootstrap3 for those who want it, and generic frontend files for those who don't, in a way that the correct
-set will automatically be chosen.
+This provides support for multiple Boilerplates, allowing you to offer rich
+frontend machinery compatible with Aldryn Boilerplate Bootstrap3 for those who
+want it, and generic frontend files for those who don't, in a way that the
+correct set will automatically be chosen.
 
-You can also add support for *other* Boilerplates, by adding the frontend files to namespaced directories in your
-application. This example of an application named `aldryn_addon` mentions only templates for sake of simplicity, but
-the same principle applies to static files::
+You can also add support for *other* Boilerplates, by adding the frontend
+files to namespaced directories in your application. This example of an
+application named `aldryn_addon` mentions only templates for sake of simplicity,
+but the same principle applies to static files::
 
     aldryn_addon
-        templates/  # the generic templates
-            aldryn_addon/
-                 base.html
```

Comment:
base should be on the templates level, aldryn_addon is just an example, but I fixed the structure